lib/Dialect/RTL/RTLFolds.cpp

```diff
@@ -17,11 +17,19 @@
 using namespace circt;
 using namespace rtl;
 
-static Attribute getIntAttr(const APInt &value, MLIRContext *context) {
+static IntegerAttr getIntAttr(const APInt &value, MLIRContext *context) {
   return IntegerAttr::get(IntegerType::get(context, value.getBitWidth()),
                           value);
 }
 
+static IntegerAttr getTrueAttr(MLIRContext *context) {
+  return IntegerAttr::get(IntegerType::get(context, 1), 1);
```

Comment:
These can also be defined as `getIntAttr(APInt(1, 1), context)` at which point it probably makes senes to inline them instead of having the getTrueAttr helpers.

Reply:
Alternatively, have one thing instead of two and use it like `getBoolAttr(true, context)`

Reply:
Deleted these